modules/editable-layers/src/edit-modes/utils.ts

```diff
@@ -516,3 +516,7 @@ export function mapCoords(
     })
     .filter(Boolean);
 }
+
+export function shouldCancelPan(event: StartDraggingEvent) {
```

Comment:
Some TSDoc here would be helpful explaining when to call and why this criteria is used.